rpcs/blockService.go

```diff
@@ -147,11 +147,22 @@ func MakeBlockService(log logging.Logger, config config.Local, ledger LedgerForB
 		memoryCap:               config.BlockServiceMemCap,
 	}
 	if service.enableService {
-		net.RegisterHTTPHandler(BlockServiceBlockPath, service)
+		service.RegisterHandlers(net)
 	}
 	return service
 }
 
+// HTTPRegistrar represents an HTTP request router that can be used by BlockService
+// to register its request handlers.
+type HTTPRegistrar interface {
```

Comment:
There is already a `Registrar` interface in this package. Have you considered expanding that instead of introducing this new one? Having both is a little confusing

Reply:
Fair, fixed